model/testresult/testresult.go

```diff
@@ -195,15 +196,38 @@ func TestResultCount(taskIds []string, testName string, statuses []string, execu
 	return Count(q)
 }
 
+// Returns tests that match exact Execution, GroupID, Statuses, TaskID, and TestName,
```

Comment:
Documentation should start with the name of the struct: `TestResultsFilterSortPaginateOpts...`